rewrite-java-core/src/main/resources/META-INF/rewrite/rewrite.yml

```diff
@@ -21,4 +27,58 @@ recipeList:
   # to io.clientcore.core.http.models.Response
   - org.openrewrite.java.ChangeType:
       oldFullyQualifiedTypeName: com.azure.core.http.rest.Response
-      newFullyQualifiedTypeName: io.clientcore.core.http.models.Response
\ No newline at end of file
+      newFullyQualifiedTypeName: io.clientcore.core.http.models.Response
+
+  # Recipe that changes all instances of com.azure.core.util.logging.ClientLogger
+  # to io.clientcore.core.util.ClientLogger
+  - org.openrewrite.java.ChangeType:
+      oldFullyQualifiedTypeName: com.azure.core.util.logging.ClientLogger
+      newFullyQualifiedTypeName: io.clientcore.core.util.ClientLogger
+
+  # Recipes to migrate implementations of com.azure.core.client.traits.HttpTrait
+  # to io.clientcore.core.models.traits.HttpTrait
+  # Add http prefix to renamed methods
+  # TODO: Filter for implementing classes
+  - org.openrewrite.java.ChangeMethodName:
+      methodPattern: com.azure.core.client.traits.HttpTrait retryOptions(..)
+      newMethodName: httpRetryOptions
```

Comment:
Doesn't this need to update the package name as well?

Reply:
The HttpTrait recipes are currently unfinished and weren't supposed to be included in this PR (were included by mistake). Removed in commit [492daeb](https://github.com/Azure/azsdk-java-rewrite-recipes/pull/14/commits/492daeb48360470829cc54218911b9b3e1758ace)

Reply:
Package name will change for all HttpTrait methods when HttpTrait is changed. This is safer to change as the last thing.